zone/guild_mgr.cpp

```diff
@@ -473,86 +504,150 @@ void ZoneGuildManager::ProcessWorldPacket(ServerPacket *pack) {
 
 	case ServerOP_GuildRankUpdate:
 	{
-		if(is_zone_loaded)
+		auto IsActionABankAction = [&](GuildAction action) -> bool {
+			std::vector<GuildAction> Guild_Bank_Actions =
+			{
+			GUILD_ACTION_BANK_DEPOSIT_ITEMS,
```

Comment:
Formatting

Reply:
Updated in next commit

Reply:
Actually had this lambda in a few spots, so converted to a regular function.